clone-install-tasks-nfsserver.yml

```diff
@@ -0,0 +1,13 @@
+---
+- import_role:
+    name: lnls-ans-role-clone-install
+  vars:
```

Comment:
@lerwys, I think we should keep cloning of web service repo and nfs repos completely separate in case they is separate  hosts in the future. As I understand, by having this playbook you clone them at the same time (even when there are separate hosts for these two services), right?

Reply:
@xresende .Yes, it makes sense. I will make the changes and separate webserv and nfsserv.